cmake/Policies.cmake

```diff
@@ -0,0 +1,14 @@
+
```

Comment:
I will add links to the corresponding CMake documentation. I added them because if you don't then on some platforms CMake will print a lot of warnings while being invoked on the CMakeLists.txt
These statements strictly define the behavior for what those rules are supposed to do in CMake and get rid of those warnings.